### PR TITLE
(maint) Set pxp-agent and mco tags for p-a 1.3.4

### PR DESCRIPTION
**Pull request by kylog**

### CODE REVIEW

**Comment by kylog** [comment]
Ping @underscorgan, @McdonaldSeanp, @stahnma. Please double-check me :)

**Comment by kylog** [comment]
Also I'll probably need to rebase after #499 is merged.

**Comment by McdonaldSeanp** [comment]
:+1: this is good =)

File: configs/components/marionette-collective.json [hunk 1]
```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/marionette-collective.git", "ref": "a28c1afe18b619c05e18c98b68007e4327fd425c"}
+{"url": "git://github.com/puppetlabs/marionette-collective.git", "ref": "refs/tags/2.8.7"}
```
Comment (joshcooper) [inline configs/components/marionette-collective.json]: why not 2.8.6?

Comment (underscorgan) [inline configs/components/marionette-collective.json]: do we not want the latest?

Comment (joshcooper) [inline configs/components/marionette-collective.json]: It looks like puppet-agent 1.3.2 shipped with mco 2.8.6, but based on https://confluence.puppetlabs.com/display/PROD/puppet-agent+Versions+and+Dependencies#puppet-agentVersionsandDependencies-puppet-agent1.3.z it looks like we do want mco 2.8.7, and there is a 2.8.7 tag, so :+1:

**Comment by kylog** [comment]
Yes 2.8.7 was motivated by requests from CS for some recent fixes.

**Pull request merged**